source/_patterns/01-molecules/content-meta/content-meta.twig

```diff
@@ -10,22 +9,21 @@
         {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-meta__item']) }) -%}
 
         <li {%- include('atoms-attributes') -%}>
-
           {%- with content only -%}
             {%- block content -%}
 
               {%- if attributes.href is defined -%}
                 {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-meta__link']) }) -%}
               {%- endif -%}
 
-              {%- include 'atoms-link' -%}
-
+              {%- include 'atoms-link' with {nodes: text} -%}
```

Comment:
This won't work: `atoms-link` uses a variable called `text`. (In turn it passes that to `atoms-text`.)